src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1068,16 +1068,20 @@ public void checkout(Run<?, ?> build, Launcher launcher, FilePath workspace, Tas
 
         try {
           checkoutCommand.execute();
-        } catch(GitLockFailedException e) {
+        } catch (GitLockFailedException e) {
```

Comment:
@abayer i can show you how deal fast with such things with sourceTree if you want :)